hi-ml-azure/run_pytest.py

```diff
@@ -59,18 +68,22 @@ class RunPytestConfig(param.Parameterized):
     )
 
 
-def run_pytest(folder_to_test: str, pytest_mark: str) -> None:
+def run_pytest(folder_to_test: str, pytest_mark: str, module_to_test: str) -> None:
     """
     Runs pytest on a given folder, restricting to the tests that have the given PyTest mark.
     If pytest finds no tests, or any of the tests fail, this function raises a ValueError. When run inside
     AzureML, this will make the job fail.
 
     :param pytest_mark: The PyTest mark to use for filtering out the tests to run.
     :param folder_to_test: The folder with tests that should be run.
+    :param module_to_test: The module for which test code coverage should be run.
     """
     results_file = Path(OUTPUT_FOLDER) / PYTEST_RESULTS_FILE
     pytest_args = [folder_to_test, f"--junitxml={str(results_file)}"]
 
+    if module_to_test:
```

Comment:
Does this mean that this argument could be an empty string? None? I think this should be documented.